nltk/model/counter.py

```diff
@@ -0,0 +1,99 @@
+# Natural Language Toolkit: Language Model Counters
+#
+# Copyright (C) 2001-2016 NLTK Project
+# Author: Ilia Kurenkov <ilia.kurenkov@gmail.com>
+# URL: <http://nltk.org/>
+# For license information, see LICENSE.TXT
+
+from __future__ import unicode_literals
+
+from collections import Counter, defaultdict
+from itertools import chain
+
+from nltk.util import ngrams
+from nltk.probability import FreqDist, ConditionalFreqDist
+from nltk import compat
+
+
+@compat.python_2_unicode_compatible
+class LanguageModelVocabulary(Counter):
```

Comment:
Doesn't this also need a membership function that returns false for terms that don't match the frequency cutoff?